contracts/HubPool.sol

```diff
@@ -184,8 +196,16 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         address originToken,
         address destinationToken
     ) public onlyOwner {
-        // TODO: Should relay message to L2 for destinationChainId and call setEnableRoute(originToken, destinationChainId, true)
         whitelistedRoutes[originToken][destinationChainId] = destinationToken;
+        relaySpokePoolAdminFunction(
+            destinationChainId,
+            abi.encodeWithSignature(
+                "setEnableRoute(address,uint32,bool)",
+                originToken,
+                uint32(destinationChainId),
```

Comment:
unrelated to this PR but I think that the unit32 is too small.

Reply:
Hm what should we use?

Reply:
EIP 155 doesn't list the highest value for this so I guess its safe to just use 256

Reply:
Will fix in #35